apps/librelingo_fakes/README.md

```diff
@@ -31,3 +31,13 @@ fake_course = fakes.customize(fakes.course1, modules=[
 ])
 ```
 
+<a name="librelingo_fakes.fakes.number"></a>
+#### number
+
+```python
+number(max_value=10000)
+```
+
+Returns some 'random' non-negatve integer.
```

Comment:
```suggestion
Returns a 'random' non-negatve integer.
```

Reply:
Also btw, regarding to this discussion, https://github.com/kantord/LibreLingo/pull/1950#discussion_r777913090
Isn't it better to decide on the edge case or just regular case if there is no edge case, to test only this case instead of having random values ?

Reply:
> Also btw, regarding to this discussion, [#1950 (comment)](https://github.com/kantord/LibreLingo/pull/1950#discussion_r777913090) Isn't it better to decide on the edge case or just regular case if there is no edge case, to test only this case instead of having random values ?

This is beyond the scope of this pull request.

Reply:
I agree with both @qbecb1zen and @vil02